src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.model.OperationModel;
+import core.basesyntax.service.FileParserService;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.ReportWriter;
+import core.basesyntax.service.service.impl.FileParserImpl;
+import core.basesyntax.service.service.impl.FileReaderImpl;
+import core.basesyntax.service.service.impl.FileWriterImpl;
+import core.basesyntax.service.service.impl.ReportWriterImpl;
+import core.basesyntax.service.strategy.TransactionStrategy;
+import core.basesyntax.service.strategy.TransactionStrategyImpl;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.List;
+
+public class Main {
+    private static final Path startPath = Paths.get("src/main/resources/inputOperation.txt");
+    private static final LocalDate localDate = LocalDate.now();
+    private static final String fileName = "src/main/resources/dailyRecord_" + localDate + ".txt";
+    private static final Path endPath = Paths.get(fileName);
```

Comment:
```suggestion
    private static final Path endPath = Paths.get("src/main/resources/dailyRecord_" + localDate + ".txt");
```
if you use one constant only in the other constant - why no merge them?

Reply:
hm, i tried but it was an error, i ll try again